src/librustc_driver/lib.rs

```diff
@@ -166,6 +166,10 @@ pub fn run_compiler<'a>(args: &[String],
 
     let sopts = config::build_session_options(&matches);
 
+    if sopts.debugging_opts.debug_llvm {
```

Comment:
Just in case something is touching LLVM before then, I didn't want to change behavior.